packages/suite-desktop/package.json

```diff
@@ -49,7 +49,7 @@
     "tinycolor2": "1.6.0",
     "tss-react": "4.9.2",
     "utif": "3.1.0",
-    "webpack": "5.95.0",
+    "webpack": "5.96.1",
```

Comment:
why webpack version needs to be changed in this PR?

Reply:
done